dev-libs/boost/boost-1.63.0.ebuild

```diff
@@ -47,6 +47,13 @@ PATCHES=(
 	"${FILESDIR}/${PN}-1.56.0-build-auto_index-tool.patch"
 	"${FILESDIR}/${PN}-1.63.0-fix-python.patch"
 )
+case ${CHOST} in
```

Comment:
You aren't really supposed to check CHOST in global-scope. Add a conditional eapply in src_prepare instead.